index.rst

```diff
@@ -28,7 +32,9 @@ Selected datasets, including both raw and prompt-processed, will be copied from
 
 Removal from the embargo repo can occur after a successful copy to normal storage, or it can be postponed for a configurable time period, but space limitations mean that no dataset should be expected to be retained for more than 30 days in the embargo repo.
 
-Handling of OUO Satellite Catalog
+Handling of the Satellite Catalog
 =================================
 
-The catalog will be treated as CUI Basic, per DOE Order 471.7. As such, access will be restricted to authorized individuals, identified in an approved acccess list and allowed access to the catalog via permissions set on a directory in the emabargo rack filesystem. The catalogs may not leave that directory, except for transient copies needed by prompt/alert processing.
+The catalog will be treated as CUI Basic, per DOE Order 471.7. As such, access will be restricted to authorized individuals, identified in an approved acccess list and allowed access to the catalog via permissions set on a directory in the embargo rack filesystem. 
```

Comment:
```suggestion
The catalog will be protected as if it was CUI Basic (as defined in [DOE Order 471.7](https://directives.dev.doxcelerate.com/directives-documents/400-series/0471.7-BOrder)). As such, access will be restricted to authorized individuals, identified in an approved access list and allowed access to the catalog via permissions set on a directory in the embargo rack filesystem. 
```

Reply:
Spotted this, @richardxdubois : since the access policy only states that the catalog "is for government official use only", and the catalog itself is not marked as CUI, I think we conclude that the catalog is potentially OUO and hence is "legacy matter", such that we need to follow Section 4.f.8 of order 471.7 (_"Legacy documents and matter that will retain Official Use Only (OUO) markings or have not been reviewed to determine if they contain CUI should also be safeguarded in accordance with the requirements established in this Directive. If safeguarding requirements cannot be met, alternate plans for appropriately protecting documents and matter that may potentially contain CUI must be developed. This alternate protection plan must be approved by the Departmental Element Designated CUI Official
or delegate."_) I'm using "protected as if it was" as shorthand for "safeguarded in accordance with" but we could use the order's wording directly.

Reply:
You could just state ts protected with out saying anything about CUI. Ed really did to want us putting the word CUI anywhere even to say it NOT CUI...

Reply:
OK - I like your suggestion below. We can show that we know what we are doing re: OUO/CUI handling in the ACP, which is not public.